src/CheckBoxFilter/Validate.ts

```diff
@@ -11,7 +11,7 @@ export class Validate {
         if (props.filterBy === "attribute" && !props.attribute) {
             errorMessage.push("Filter by 'Attribute' requires an 'Attribute'");
         }
-        if (props.filterBy === "attribute" && !props.attributeValue) {
+        if (props.filterBy === "attribute" && !props.attributeValue.trim()) {
```

Comment:
why is there a trim needed? What if the use like to search for " hello "

Reply:
Removed